extensions/amp-accordion/1.0/test-e2e/e2e-fixture.js

```diff
@@ -0,0 +1,6 @@
+import * as Preact from '#preact';
```

Comment:
Can this file be autogenerated as well?

Reply:
I support this. This can be a constant file that just gets copied without any changes.

There's an inconsistency in the `_default` name across the codebase, but you could do something like:

```js
import {_default, Default} from '../storybook/Basic';
const App = Default || _default;
```

Reply:
I'm having trouble thinking about how to generate this file, so I think I'm not going to for right now.

I wanted to try and have this entry point just loaded in memory, but I can't figure out if esbuild supports that. Alternatively, I didn't want to have this file get generated and need to be committed, and adding this file to a temporary location, like a dist directory inside the extensions directory, didn't sit quite right with me either since this isn't really a distribution file.

Anyway, we can revisit generating this later.

Reply:
@alanorozco The linter does not like having an import that doesn't exist. So I don't think your code snippet will work.